dnsvizor.opam

```
@@ -26,6 +26,7 @@ depends: [
   "cmdliner" {>= "1.2.0"}
   "logs" {>= "0.7.0"}
   "ohex" {>= "0.2.0"}
+  "charrua"
```

```suggestion
  "charrua" {>= "2.1.0"}
```

That's just a guess, since the unikernel requires 2.1.0, I don't see any loss in putting such a lower bound.

---

What do you think about this suggestion?